src/lib.rs

```diff
@@ -141,9 +140,9 @@ impl Clipboard {
                 .zip(start_time)
                 .next()
                 .map(|(timeout, time)| (Instant::now() - time) >= timeout)
-                .unwrap_or(false)
+                .is_some()
```

Comment:
I think that `.is_some()` cannot replace `.unwrap_or(false)`.

Reply:
You're right, in the case that it's `Some(false)`. Thanks.